### ISSUE TITLE
Map shows some orange dots

### ISSUE THREAD

**Issue by pbb72** [mapillary/mapillary_issues#2109]
The map shows, among green areas, several red/orange dots with white outlines. A few dots are bigger than others. Hovering over them, shows a photo with an orange Mapillary logo.

My initial thought was that these dots just represent capital cities on the map. But that didn't make sense. The fact that some of these dots are bigger than others makes things more puzzling.

After checking several of the photos, I start recognizing some of them from the Chrome new tab plugin. My guess these are "featured" photos, but there is not feedback anywhere that confirms this.

This needs to be more obvious. Preferably in the map view (so that the meaning is obvious without any further actions from the user) but an alternative is some hint to this in the hover thumbnail (not ideal since this requires action from the user to understand the meaning, and an orange logo is not clear enough).

**Comment by angelday** [comment]
Yes, Mapillary featured photos are marked with an orange sigil on hover. We also had designs in place to explain what these featured places are and eventually we'll improve the popup for featured markers. 👍

**Comment by gyllen** [comment]
@millenbop Don't forget LABEL guidelines here add either `feature`, `discussion` or `bug`

**Comment by talllguy** [comment]
@gyllen 🚓  🚓  🚓  🚓  😂

**Comment by gyllen** [comment]
@talllguy Thats right

/ the github police

**Comment by pbb72** [comment]
:scream:
